src/transfers/transfers.service.ts

```diff
@@ -0,0 +1,106 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Cron } from '@nestjs/schedule';
+import { getExtrinsicsStat } from './transfers.queries';
+import { TransfersStats } from '../../common/entities/TransfersStats';
+import { GqlService } from '../../common/gqlService';
+import { AppConfigService } from '../utils/app-config.service';
+import { UtilsService } from '../utils/utils.service';
+
+enum ExtrinsicsStatsTypeEnum {
+  COINS = 'COINS',
+  TOKENS = 'TOKENS',
+  TRANSACTIONS = 'TRANSACTIONS',
```

Comment:
Why do we have three types in this repo, but only two in `unique-scan-backend` repo (COINS and TOKENS)? TRANSACTIONS means everything?

Reply:
yep